src/cascadia/LocalTests_TerminalApp/TerminalApp.LocalTests.vcxproj

```diff
@@ -9,6 +9,7 @@
     <ConfigurationType>DynamicLibrary</ConfigurationType>
     <WindowsTargetPlatformMinVersion>10.0.18362.0</WindowsTargetPlatformMinVersion>
     <WindowsTargetPlatformVersion>10.0.18362.0</WindowsTargetPlatformVersion>
+    <OpenConsoleUniversalApp>true</OpenConsoleUniversalApp>

```

Comment:
Uh, what's keying on this? Did I miss it?

Reply:
That's a good question. I think I had to add this when I made the project a winrt project, but maybe I didn't need to. Lemme check.

Reply:
Yep, didn't need this.